extractor/src/main/java/org/schabi/newpipe/extractor/comments/CommentsInfo.java

```diff
@@ -86,7 +92,6 @@ public void setCommentsExtractor(final CommentsExtractor commentsExtractor) {
     }
 
     /**
-     * @apiNote Warning: This method is experimental and may get removed in a future release.
```

Comment:
I don't think that was the reason. The `Optional` was in YouTube's specific implementation, not here. Anyway, I think it does not make sense to keep this as experimental. It has not been changed [since one and a half year](https://github.com/TeamNewPipe/NewPipeExtractor/commit/fdebf3c6cdf88fed8a372d7d7376acfa3a9f8c10) and will probably not be changed again any time soon. So this is as experimental as all other extractor features, which may change in case we need to change it, but generally things don't change.